### ISSUE TITLE
Noob question

### ISSUE THREAD

**Issue by taigams** [SLScriptersGroup/SL-RP-HUD#14]
Hello, i'm a student and i have only a small doubt. Is there any way i can get in touch with you? i'm a noob. I'm studing about coding and i'm a second life player so i saw your project and i was reading about this. I need to have a VPS for this system to work or can be a simple webhost?

**Comment by SylasSeabrook** [comment]
This project is not one for those who are not already very familiar with the technologies involved. To get started with SL's LSL, I highly recommend picking something small that's already being done by someone and trying to recreate it. As you familiarize yourself with LSL, you'll find many more opportunities for exercising and expanding your skills.

**Comment by taigams** [comment]
Yes yes, I will do that. I just want to know for  knowledge, to understand what is needed. Could you please inform me?

**Comment by SylasSeabrook** [comment]
I'm not going to answer that. By the time you get back here with some background, that question's answer will be obvious. Start your journey at the beginning. :)

**Comment by taigams** [comment]
Wow, you don't have to be rude. I was just trying to clear up a doubt, to understand how things work. You don't really have to be rude, but yeah, thanks anyway. Good luck to you
